index.html

```diff
@@ -38,146 +38,138 @@
-        <h3 class="fw9 f3 w-80 w-100-ns center tl">Hands-on learning to Empower and Inspire.</h3>
-        <p class="w-100-ns center tl f5 b">Join one of our masterclasses or tailor a class to suit your needs</p>
-        <ul class="b pl3 center tl" style="font-size: 0;">
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">Agile Development Methodology</li>
-            <li class="mb1 pr2">Customer Development</li>
-            <li class="mb1 pr2">Cyber Security</li>
-          </div>
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">User Testing</li>
-            <li class="mb1 pr2">Product Ownership</li>
-            <li class="mb1 pr2">Advanced Analytics</li>
-          </div>
-          <div class="dib-ns w-third-ns f5 v-top">
-            <li class="mb1 pr2">Human-centered Design</li>
-            <li class="mb1 pr2">Code</li>
-            <li class="mb1 pr2">Data Visualisation</li>
-          </div>
-        </ul>
-      </div>
-      <div class="tc mt4-ns center mb5 mb5-l">
-        <a href="#contact" class="dib center link mt2 ph4 pv2 dwyl-bg-mint white br2 f5 shadow-4">Enquire Now</a>
+    <section class="w-100 dwyl-bg-dark-gray tc tl-ns mw8 center pt4 ph5">
+      <div class="mw6-5 center">
+        <h2 class="white text-shadow f2 mv0 pt4">Training &amp; Workshops</h2>
+        <div class="dwyl-dark-gray">
+          <h3 class="fw9 f3 dwyl-yellow center tl mb0 m5">Hands-on learning to Empower and Inspire.</h3>
+          <p class="w-100-ns center tl f5 white b mt2">Join one of our masterclasses or tailor a class to suit your needs</p>
+          <ul class="b center tl pl0 mt3" style="font-size: 0;">
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Customer Development</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Cyber Security</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">User Testing</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Product Ownership</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Advanced Analytics</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Human-centered Design</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Agile Development Methodology</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Code</li>
+            <li class="v-top dib w-100 w-50-ns f5 dwyl-yellow fw3 mb3 pb2 pr3">Data Visualisation</li>
+          </ul>
+        </div>
+        <a href="#contact" class="dib center link mt2 mb5 ph4 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">Enquire Now</a>
       </div>
     </section>
 
-    <section class="dwyl-bg-mint center pv5-ns pb5-ns relative z-0">
+  </main>
+
+    <section class="dwyl-bg-mint center pv5-ns pb5-ns relative z-0 mw9">
```

Comment:
Not entirely convinced by this section being boxed in by the sides, I feel like a footer should cover the expanse of the webpage, but not offended enough to ask for the change:
<img width="1258" alt="screen shot 2017-10-09 at 19 17 08" src="https://user-images.githubusercontent.com/4185328/31352395-92a70c7a-ad26-11e7-8e8d-30eca085cdf4.png">

It does need to be consistent across the board however (see `404.html`) and I think that would be easier if the Training box was enclosed within the white 'Services' box (essentially) and this expanded across the site.

Reply:
I've expanded the footer and the training section:

![selection_097](https://user-images.githubusercontent.com/22300773/31382287-0c8fd098-adaf-11e7-8d87-487c87867378.png)